dotCMS/src/main/java/com/dotmarketing/image/focalpoint/FocalPointAPIImpl.java

```diff
@@ -57,12 +60,14 @@ public void writeFocalPoint(final String inode, final String fieldVar, final Foc
           return;
         }
 
-        if(tempFileAPI.isTempResource(inode)){
+        if(tempFileAPI.isTempResource(inode) || isSet(inode) && inode.startsWith(TMP)) {
```

Comment:
Is it possible to create an IT over this validation?

Reply:
done.